storybook/src/elements/actions/actions.stories.js

```diff
@@ -0,0 +1,209 @@
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-add-to-collection-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-favorites-toggle-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-delete-blob-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-delete-document-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-download-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-export-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-link-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-lock-toggle-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-move-documents-down-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-move-documents-up-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-notifications-toggle-button';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-share-button.js';
+import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-untrash-document-button.js';
+
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { color } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-icons';
+import fakeServer from '../../mock/nuxeo.mock';
+import { DOCUMENT1, DOCUMENT2, DOCUMENTS, DOCUMENT_DOWNLOAD, DOCUMENT_TRASHED } from '../../mock/documents.mock';
```

Comment:
Small detail: if we are importing `DOCUMENTS` that is an array containing the three documents, we may not need to import also `DOCUMENT1` and `DOCUMENT2`, since we could just do something like `DOCUMENTS[DOCUMENT1] ` or `DOCUMENTS[index]`.

Anyway, it is just a suggestion to think further on how we want to represent mocks. Obviously it would need imply to have a stronger semantic, since DOCUMENT`NUMBER` may not be relevant.

Reply:
Will do

Reply:
Just a suggestion, but if you find it helpful 👍

Reply:
You can namespace it as well:
```suggestion
import * as DOC_MOCKS from '../../mock/documents.mock';
```
And then use `DOC_MOCKS.DOCUMENT1`

Reply:
Do you think this way is more meaningful than using `DOCUMENTS[1]`?